tests/run-make/pgo-branch-weights/main.rs

```diff
@@ -1,7 +1,7 @@
 extern crate interesting;
 
 fn main() {
-    let arg = std::env::args().skip(1).next().unwrap();
+    let arg = "aaaaaaaaaaaa2bbbbbbbbbbbb2bbbbbbbbbbbbbbbbcc";
```

Comment:
I think this *needs* to be an env var to inhibit optimizations?

Reply:
The `run` function in `run.rs` might need to be restructured significantly to accept arguments and not just a path to a file. I've just tried lazily passing the arg in the most recent commit, but I'm not hopeful.